Production Releasing.md

```diff
@@ -6,26 +6,34 @@ Not all checklist items are necessary in every app.
 
 If your not sure whether you should add some feature (e.g. Google Analytics) to the webapp your releasing, consult with the project manager and/or your team leader.
 
+In your project management app (e.g. Productive, Trello...) create a task _Production Checklist_ and link to this handbook chapter. Finish it prior to production releasing.
+
 ## Checklist items:
 
-* Test the app on various browsers. The usual set of browsers is (this can vary from project to project): Chrome, FireFox, Opera, Safari, IE8+. HTML apps only.
-* Go through a final design review with the designer. HTML apps only.
-* Handout the application to the QA team so they can try to find potential bugs before the client or customers do. HTML apps only.
 * Are you uploading all files to S3 and not to the local (public/system) filesystem?
-* Did you add your application to the [Application list](https://github.com/infinum/internal-guides/blob/master/applications/Readme.md)
-* Did you add monitoring via UptimeRobot?
-* If the project is on code climate, check that it doesn't have any security issues. If the project isn't on code climate, check the security of your application with [Brakeman](https://github.com/presidentbeef/brakeman)
-* Where's the DNS zone? If there are no dependencies (like Cpanel, lots of subdomains etc..), you're probably better off hosting it on our DNS servers (PointHQ) because it's faster to do changes then contacting a third party DNS hoster
-* Lower TTL for all A/CNAME records to 300.
-* Are you using SSL?
-* Add the cookies_eu gem.
+* Check that the app doesn't have any security issues on Code Climate. If it isn't on Code Climate, why isn't it? If there is a explicit reason why it isn't, check the security of your application with [Brakeman](https://github.com/presidentbeef/brakeman)
 * Add bugsnag.
+* Gzip the content with [Rack Deflater](http://robots.thoughtbot.com/content-compression-with-rack-deflater/)
+* Check if indices are all setup - if you have the postgres `pg_stat_statements` extension enabled, check what are some of the slow queries and consult with the team how to decrease their execution time. Also checkout [lol_dba](https://github.com/plentz/lol_dba) - a static checker for indices based on foreign keys.
```

Comment:
Just a heads up for this, we are currently testing [pghero](https://github.com/ankane/pghero/) which shows suggested indexes based on actually ran queries :). Also does a static check on indexes. For example: on truck it suggested that i deleted a index, because it already was included in another compound index ![](https://scr.infinum.co/stjepan_hadjic/PgHero__Overview_2017-04-14_08-48-56.png)